server/src/internalClusterTest/java/org/opensearch/search/aggregations/EquivalenceIT.java

```diff
@@ -195,6 +195,7 @@ public void testRandomRanges() throws Exception {
             reqBuilder = reqBuilder.addAggregation(filter("filter" + i, filter));
         }
 
+        verifyStoreContent();
```

Comment:
On actual test report, there was one failure `testDuelDepthBreadthFirst` for which I don't see the fix here ?